src/components/TableComponents/TicketsBodyComponents/TicketsBodyComponents.tsx

```diff
@@ -0,0 +1,54 @@
+import { TableBody, TableCell } from "@mui/material";
+
+import CellTextItem from "../CellTextItem";
+import { TICKETS_DATA } from "../../../constants";
+import { ReactComponent as MoreIcon } from "../../../assets/logos/more.svg";
+
+import { BodyRow, UserBox, UserPhoto } from "./styled";
+import { priorityMarkers } from "./helper";
+
+interface ITicketsBodyComponents {
+  page: number;
+  rowsPerPage: number;
+}
+
+const TicketsBodyComponents = ({
+  page,
+  rowsPerPage,
+}: ITicketsBodyComponents) => {
+  return (
+    <TableBody>
+      {TICKETS_DATA.slice(
+        page * rowsPerPage,
+        page * rowsPerPage + rowsPerPage
+      ).map((row) => (
+        <BodyRow key={`${Math.random()} ${Math.random()}`}>
```

Comment:
Never use random in id's or key's. Its not safe as you don't have any proof that you wont have a few similar values

Reply:
done